lib/course_clone_manager.rb

```diff
@@ -3,10 +3,11 @@
 
 #= Procedures for creating a duplicate of an existing course for reuse
 class CourseCloneManager
-  def initialize(course, user, campaign_slug=nil)
+  def initialize(course, user, clone_assignments, campaign_slug=nil)
```

Comment:
I think this would benefit from converting the positional arguments to named arguments. That will make it more clear what all the parameters are, whenever this method is called.